File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldLanguage.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.LanguageKey;
import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldLanguage extends DefnFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public LanguageKey defaultValue;

  @Nullable
  public LanguageKey defaultVar;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldLanguage.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.LanguageKey
import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldLanguage : DefnFieldEditable
{
  val defaultFieldId: MetaIdField?
  val defaultValue: LanguageKey?
  val defaultVar: LanguageKey?
}